origins_qa/origins_qa.routing.yml

```diff
@@ -34,3 +34,16 @@ origins_qa.manager.qa_account_create_form_modal:
     _permission: 'manage qa accounts'
   options:
     _admin_route: TRUE
+
+origins_qa.api.users_enable:
+  path: '/origins-qa/api/users/enable/{token}'
```

Comment:
Maybe restrict what HTTP methods can be used here too? HTTP POST might be the only thing you need?

Reply:
Great idea.